elements/pfe-accordion/src/pfe-accordion.js

```diff
@@ -258,17 +254,21 @@ class PfeAccordion extends PFElement {
   }
 
   _animate(panel, start, end) {
-    const header = panel.previousElementSibling;
-    panel.classList.add("animating");
-    header.classList.add("animating");
-    panel.style.height = `${start}px`;
+    if (panel) {
```

Comment:
same w/ this file, is this just a merge issue, seems out of scope for this branch

Reply:
Same here